packages/graphql_client/lib/src/contracts.dart

```diff
@@ -0,0 +1,225 @@
+import 'package:graphql_client/graphql_client.dart';
+import 'package:graphql_client/models.dart';
+
+class TFContracts {
+  final GraphQLClient gqlClient;
+
+  TFContracts(this.gqlClient);
+
+  Future<List<NameContract>> listNameContractsByTwinId(
```

Comment:
why list the contracts only by twinId? I think we have some query options to be supported too

Reply:
please check that all methods have query options (including limit, offset, and order)